woodwork/serialize_accessor.py

```diff
@@ -0,0 +1,180 @@
+import datetime
+import json
+import os
+import tarfile
+import tempfile
+
+import pandas as pd
+
+import woodwork as ww
+from woodwork.s3_utils import get_transport_params, use_smartopen
+from woodwork.type_sys.utils import (
+    _get_ltype_class,
+    _get_specified_ltype_params
+)
+from woodwork.utils import _is_s3, _is_url, import_or_none
+
+dd = import_or_none('dask.dataframe')
+ks = import_or_none('databricks.koalas')
+
+SCHEMA_VERSION = '6.0.0'
+FORMATS = ['csv', 'pickle', 'parquet']
+
+
+def typing_info_to_dict(dataframe):
+    '''Gets the description for a Woodwork table, including typing information for each column
+    and loading information.
+    '''
+    ordered_columns = dataframe.columns
+    column_typing_info = [
+        {'name': col_name,
+         'ordinal': ordered_columns.get_loc(col_name),
+         'logical_type': {
+             'parameters': _get_specified_ltype_params(col['logical_type']),
+             'type': str(_get_ltype_class(col['logical_type']))
+         },
+            'physical_type': {
```

Comment:
Extra indent here

Reply:
unindented this and the couple of fields below that are also too far indented